WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -432,11 +437,16 @@ def execute( self ):
     checkCmd = 'dirac-wms-get-wn-parameters -S %s -N %s -Q %s %s' % ( self.pp.site, self.pp.ceName, self.pp.queueName,
                                                                        " ".join( self.cfg ) )
     retCode, result = self.executeAndGetOutput( checkCmd, self.pp.installEnv )
-
-    result = result.split( ' ' )
-    numberOfProcessor = int( result[0] )
-    maxRAM = int( result[1] )
-
+    if retCode:
+      self.log.error( "Could not get resource parameters [ERROR %d]" % retCode )
+      self.exitWithError( retCode )
+    try:
+      result = result.split( ' ' )
```

Comment:
I think here it should be rather just result = result.split(), we need here splitting by white spaces whatever the number of those